MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,113 +1,64 @@
-        let text: String
-        let buttonText: String
-    }
-
-    private let questions: [QuizQuestion] = [
-        QuizQuestion(
-            image: UIImage(named: "The Godfather"),
-            rating: 9.2,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: true),
-        QuizQuestion(
-            image: UIImage(named: "The Dark Knight"),
-            rating: 9,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: true),
-        QuizQuestion(
-            image: UIImage(named: "Kill Bill"),
-            rating: 9.2,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: true),
-        QuizQuestion(
-            image: UIImage(named: "The Avengers"),
-            rating: 8,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: true),
-        QuizQuestion(
-            image: UIImage(named: "Deadpool"),
-            rating: 8,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: true),
-        QuizQuestion(
-            image: UIImage(named: "The Green Knight"),
-            rating: 6.6,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: true),
-        QuizQuestion(
-            image: UIImage(named: "Old"),
-            rating: 5.8,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: false),
-        QuizQuestion(
-            image: UIImage(named: "The Ice Age Adventures of Buck Wild"),
-            rating: 4.3,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: false),
-        QuizQuestion(
-            image: UIImage(named: "Tesla"),
-            rating: 5.1,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: false),
-        QuizQuestion(
-            image: UIImage(named: "Vivarium"),
-            rating: 5.8,
-            question: "Рейтинг этого фильма больше чем 6?",
-            answer: false)
-    ]
-
+final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
     private var currentQuestionIndex = 0
     private var score = 0
-    private var totalScore = 0
-    private var highestScore = 0
-    private var highestScoreDate = ""
-    private var quizesPlayed = 0
-    private var averageAccuracy: Float = 0
     private let dateFormatter = DateFormatter()
     private let dateFormat = "dd.MM.yy HH:mm"
-
+    private var statisticService: StatisticService?
+    private let questionsAmount: Int = 10
+    private var questionFactory: QuestionFactoryProtocol?
+    private var currentQuestion: QuizQuestion?
     @IBOutlet private var imageView: UIImageView!
     @IBOutlet private var counterLabel: UILabel!
     @IBOutlet private var textLabel: UILabel!
     @IBOutlet private weak var yesButton: UIButton!
     @IBOutlet private weak var noButton: UIButton!
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
```

Comment:
```suggestion
    @IBOutlet private weak var activityIndicator: UIActivityIndicatorView!
```

Давайте всегда выставлять самый закрытый уровень доступа по умолчанию 🙏